### PR TITLE
sci-mathematics/calc: add 2.14.1.6

### PR DESCRIPTION
**Pull request by jubalh**

### CODE REVIEW

File: sci-mathematics/calc/calc-2.14.1.6.ebuild [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+		LDFLAGS="${LDFLAGS}" \
+		EXTRA_LDFLAGS="${LDFLAGS}" \
+		CALCPAGER="${PAGER}" \
+		USE_READLINE="-DUSE_READLINE" \
+		READLINE_LIB="-lreadline -lhistory $($(tc-getPKG_CONFIG) --libs ncurses) -L\"${S}\"/custom -lcustcalc" \
+		all
+}
+
+src_test() {
+	if echo "${LD_PRELOAD}" | grep -q "sandbox"; then
```
Comment (thesamesam) [inline sci-mathematics/calc/calc-2.14.1.6.ebuild:47]: Is this still right? This sounds kind of suspicious.

File: sci-mathematics/calc/calc-2.14.1.6.ebuild [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (thesamesam) [inline sci-mathematics/calc/calc-2.14.1.6.ebuild:4]: 8 pls

File: sci-mathematics/calc/calc-2.14.1.6.ebuild [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+DESCRIPTION="Arbitrary precision C-like arithmetic system"
+HOMEPAGE="http://www.isthe.com/chongo/tech/comp/calc/"
+SRC_URI="http://www.isthe.com/chongo/src/calc/${P}.tar.bz2"
+
+SLOT="0"
+LICENSE="LGPL-2"
+KEYWORDS="~alpha ~amd64 ~arm64 ~ppc ~ppc64 ~x86 ~amd64-linux ~x86-linux"
+
+RDEPEND="
+	sys-libs/ncurses:0=
```
Comment (thesamesam) [inline sci-mathematics/calc/calc-2.14.1.6.ebuild:17]: :0= => :=

File: sci-mathematics/calc/files/calc-2.14.1.6-RPATH.patch [hunk 4]
```diff
@@ -0,0 +1,87 @@
+diff -urEbw calc-2.14.1.6/Makefile calc-2.14.1.6.new/Makefile
```
Comment (thesamesam) [inline sci-mathematics/calc/files/calc-2.14.1.6-RPATH.patch:1]: Run iwdevtools' scrub-patch on this

**Comment by thesamesam** [comment]
I get this:
```

 * QA Notice: Files built without respecting LDFLAGS have been detected
 *  Please include the following list of files in your report:
 * /usr/lib64/libcalc.so.2.14.1.6
 * /usr/lib64/libcustcalc.so.2.14.1.6
```

See https://gist.github.com/thesamesam/1a48cee5cc9584dd70604be22e544557.

**Comment by jubalh** [comment]
I'll give up on this one.

**Pull request closed without merging**